src/kOS/Binding/FlightControlManager.cs

```diff
@@ -45,176 +41,80 @@ public override void AddTo(SharedObjects shared)
             SafeHouse.Logger.Log("FlightControlManager.AddTo " + Shared.Vessel.id);
 
             currentVessel = shared.Vessel;
-            ConnectivityManager.AddAutopilotHook(currentVessel, OnFlyByWire);
 
-            AddNewFlightParam("throttle", Shared);
-            AddNewFlightParam("steering", Shared);
-            AddNewFlightParam("wheelthrottle", Shared);
-            AddNewFlightParam("wheelsteering", Shared);
+            shared.BindingMgr.AddBoundVariable("THROTTLE", GetThrottleValue, SetThrottleValue);
+            shared.BindingMgr.AddBoundVariable("STEERING", GetSteeringValue, SetSteeringValue);
+            shared.BindingMgr.AddBoundVariable("WHEELSTEERING", GetWheelSteeringValue, SetWheelSteeringValue);
+            shared.BindingMgr.AddBoundVariable("WHEELTHROTTLE", GetSteeringValue, SetSteeringValue);
```

Comment:
@Dunbaratu This would be why you got a steering error for `wheelthrottle`, copy/paste error on my part.  I can fix it on my end or you you can fix it on your end, whichever is easier.

Reply:
I just mentioned in in a review comment (just published a minute ago before I read this).  But I didn't change it.